packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -47,7 +49,13 @@ class AndroidStudio implements Comparable<AndroidStudio> {
     Version version;
     if (versionString != null)
       version = Version.parse(versionString);
-    return AndroidStudio(studioPath, version: version);
+
+    final String plistValue = iosWorkflow.getPlistValueFromFile(
+      plistFile,
+      null,
+    );
+    final String pathsSelectorValue = _pathsSelectorMatcher.stringMatch(plistValue).split('=').last.trim().replaceAll('"', '');
```

Comment:
@chinmaygarde, somewhat tangential but you added a note to not use PlistBuddy in #1969 but we still have a wrapper in https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/ios/mac.dart#L38. Should we remove it?